src/main.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class main {
+
+    public static void main(String[] args) {
+        ArrayList<String> products = new ArrayList<String>();
+        Scanner scan = new Scanner(System.in);
+        System.out.println("do you want start?(yes)");
+        String anserw0 = scan.next();
+        while (anserw0.equals("yes")) {
+            System.out.println("pls enter product name");
+            products.add(scan.next());
+
+            System.out.println("do you want add next products ? yes/no");
+            String anserw = scan.next();
+
+            if (anserw.equals("no")) {
+                System.out.println("thanks for focus " + products);
+                break;
+
+            }
+        }
+        System.out.println("do you want go to step 2? enter yes");
+        String anserwer = scan.next();
+        while (anserwer.equals("yes")) {
```

Comment:
If u use switch statement it will probably require only one while loop for whole code.